hangman.rb

```diff
@@ -0,0 +1,38 @@
+# Unused code This class was my original way of doing the project
+
+# class HangMan
+#     def initialize(word, guess)
+#         @word = word.downcase
+#         @guess = guess
+#         @word_array = []
+#     end
+
+#     def guess
+#         @word.each_char do |c| 
+#             if @guess.include?(c)
+#                 @word_array.push(c)
+#             else
+#                 @word_array.push("_")
+#             end
+#         end
+#         puts
+#         puts @word_array.join()
+#     end
+# end
+
+# hang = HangMan.new("There", ["e", "h"])
+# hang.guess()
+
+def hangman(secret_word, guess)
+    word_array = []
```

Comment:
Alternatively, you could define an empty string, keep adding to it (e.g., `wordString += c`), and then return it directly to avoid the performance overhead from the Array->String conversion on the `word_array.join`, but what you have still completes the assignment just fine. Good job!